packages/components/src/components/post-icon/post-icon.tsx

```diff
@@ -41,7 +45,7 @@ export class PostIcon {
   /**
    * The base path, where the icons are located (must be a public url).
    */
-  @Prop() readonly base?: string;
+  @Prop() readonly base?: string = '';
```

Comment:
Technically you're totally right.
But from the users view, it would make more sense to me, to only use the property for overwriting the defaultvalue (which is stored internally).